array.md

```diff
@@ -0,0 +1,42 @@
++ [Squares of a Sorted Array](#squares-of-a-sorted-array)

```

Comment:
VERIFICATION RESULT : 
 Hello! 
 I am here to help you with making your pr title looks better. Now it is: 
 added new file 
 Mistakes: 
 Dude...it is not even in right format.